core/typedefs.h

```diff
@@ -315,4 +315,6 @@ struct BuildIndexSequence<0, Is...> : IndexSequence<Is...> {};
 #define ___gd_is_defined(val) ____gd_is_defined(__GDARG_PLACEHOLDER_##val)
 #define GD_IS_DEFINED(x) ___gd_is_defined(x)
 
+#define FORCE_SEMICOLON ;
```

Comment:
You forgot to remove this before merge.

We might want to sneak this into another codebase wise refactoring, as changing `typedefs.h` again will cause a recompiling of all artifacts for contributors and CI.

Reply:
Incorporated the macro removal in #93401

Reply:
> You forgot to remove this before merge.

Sorry, that's my fault!